### PR TITLE
Removed in-game references to clowns.

### PR DESCRIPTION
**Pull request by Kaedwuff**
Several everyday items referenced clowns offhandedly, even though we no longer have them on server. This has been fixed.

All the clown items still exist, though, because they are just clown themed items, not unrelated things containing clown jokes.

Feedback Topic: https://forums.aurorastation.org/viewtopic.php?f=18&t=11347&sid=2d6b34426f7b5e2be25d852fc8f0e661

### CODE REVIEW

**Comment by skull132** [comment]
This is how culture dies.

**Comment by ParadoxSpace** [comment]
'Removes fun'

**Comment by HeyBanditoz** [comment]
Why tho
Do you guys still have the clown clothes and horn and stuff?

**Comment by Ornias1993** [comment]
No clown best clown!

File: code/game/objects/items/weapons/material/knives.dm [hunk 1]
```diff
@@ -79,6 +79,6 @@
 	name = "butcher's cleaver"
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "butch"
-	desc = "A huge thing used for chopping and chopping up meat. This includes clowns and clown-by-products."
```
Comment (Ornias1993) [inline code/game/objects/items/weapons/material/knives.dm]: I actually think this one is okey, it doesnt really reference "THe clown" just clowns and doesnt everyone want to chop up clowns?

**Comment by OneOneThreeEight** [comment]
The only thing I thought as of recent was any kind of problematic was the IC reference to clowns in the PDA server passkey document. That's sort of the only thing that really mattered.

**Comment by NiftyBaby** [comment]
Kaed hates fun.

On Wed, May 23, 2018 at 10:08 AM, OneOneThreeEight <notifications@github.com

**Comment by Ornias1993** [comment]
@OneOneThreeEight Well the portal thing (although not as frequently spotted) is abitt odd too... But I agree... Funny references to clowns aren't bad, but too much focuss on them or "the clown" feels out of place in the current state of aurora.

**Comment by Kaedwuff** [comment]
Any news on this being merged? It's been approved :V

**Comment by HeyBanditoz** [comment]
I wonder why it hasn't been merged yet.... 🤔🤔🤔🤔🤔🤔🤔🤔
:thinking:

**Comment by HeyBanditoz** [comment]
Also given Aurora's hard on for suggestions shouldn't there be a thread 
on this? It's obviously universally disliked.

**Comment by ParadoxSpace** [comment]
yeah literally no one wants this

**Comment by Arrow768** [comment]
I´ve deleted a few responses from that PR.
Please remain civilized.

**Comment by Arrow768** [comment]
A feedback topic is now available:
Please provide any feedback regarding the PR there:
https://forums.aurorastation.org/viewtopic.php?f=18&t=11347&sid=2d6b34426f7b5e2be25d852fc8f0e661

**Comment by Ornias1993** [comment]
@Kaedwuff it was approved code and gameplay wise... that doesn't mean it's a wanted change ;)

**Comment by Arrow768** [comment]
No negative feedback has been provided in the feedback topic.
In it goes.

**Pull request merged**